_posts/2021-05-05-the-competitive-advantage-of-ux.md

```diff
@@ -0,0 +1,78 @@
+
+Based on this, I believe value and subsqeuent competitive advantage is greatly enabled by understanding how to optimise the relationship between UX and DevOps.
+<br /><br />
+
+### 1. Discovery
+
+At Mechanical Rock before we start a piece of design work, we run in depth discovery sessions. Developers, designers and the clients are involved. In these sessions, we understand the problem space, dive deeper into the current state and user journey. A shared understanding is fostered as both designers and developers have the same context of the problem space.
+
+Without this session, it is too easy to make assumptions and be driven by the client leading to poor delivery. For example, on a recent project, the client was adamant that the users felt fearful of artifical intelligence and machine learning; however, our user research suggested otherwsie, which opened up new opportunities and allowed us to deliver solutions driven by user data. This is an example, of how UX provides visibility into the problem space and user needs which leads to strong delivery.
+
+![Cypress CodeBuild report](/img/Quickmail_Discovery.jpg)
+_Karen Cheng, Principal Design Consultant, running a discovery workshop with a client_
+
+<br /><br />
+
+### 2. Removal of Silos
+
+DevOps is about removing waste to support the flow of value through a system. This is enabled by the removal of silos, feedback loops and a shared understanding. Value in DevOps concerns whether features add value or are waste. Waste being a redundant functionality that is not useful. The flow of value through a system is impeded by constraints such as silos.
+
+Within product teams, silos occur when different specialisations work separately. Teams work in a vacuum, doing only their part without wider considerations, constraints, ideas and critiques of the larger team. For example, when development is siloed from UX, designers do not have visibility into software constraints, and therefore the system in which a user experience operates in. Decisions are made with partial context; impeding this flow and therefore value. Considering UX offers a long term competitive advantage “it’s important to understand these relationships from a UX perspective because they illuminate how interactions between enterprise systems can exert significant impacts on the user experience”, ([Amy Chess](https://www.uxmatters.com/mt/archives/2019/03/platform-ux-domain-driven-design-concepts-in-enterprise-ux.php)). Removal of silos allows continuous feedback loops and a shared understanding to foster between designers and developers which removes constraints and enables flow. Without that it is no different to pre-DevOps days.
```

Comment:
```suggestion
Within product teams, silos occur when different specialisations work separately. Teams work in a vacuum, doing only their part without wider considerations, constraints, ideas and critiques of the larger team. For example, when development is siloed from UX, designers do not have visibility into software constraints, and therefore the system in which a user experience operates in. Decisions are made with partial context; impeding this flow and therefore value. Considering UX offers a long term competitive advantage: “it’s important to understand these relationships from a UX perspective because they illuminate how interactions between enterprise systems can exert significant impacts on the user experience”, ([Amy Chess](https://www.uxmatters.com/mt/archives/2019/03/platform-ux-domain-driven-design-concepts-in-enterprise-ux.php)). Removal of silos allows continuous feedback loops and a shared understanding to foster between designers and developers which removes constraints and enables flow. Without that it is no different to pre-DevOps days.
```

Reply:
I don't know what this change was!  Or you've already made it....